lib/semian/simple_exponential_smoother.rb

```diff
@@ -0,0 +1,131 @@
+# frozen_string_literal: true
+
+module Semian
+  # SimpleExponentialSmoother implements Simple Exponential Smoothing (SES) for forecasting
+  # a stable baseline error rate in adaptive circuit breakers.
+  #
+  # SES focuses on the level component only (no trend or seasonality), using the formula:
+  #   smoothed = alpha * value + (1 - alpha) * previous_smoothed
+  #
+  # Key characteristics:
+  # - Low alpha provides low recency bias, making it resilient to short incidents
+  # - Gradually adapts to sustained changes in error rate
+  # - Clips extreme values to prevent outliers from distorting the forecast
+  #
+  # Example usage:
+  #   smoother = SimpleExponentialSmoother.new
+  #   smoother.add_observation(0.01)
+  #   baseline = smoother.forecast  # Returns smoothed baseline error rate
+  #
+  class SimpleExponentialSmoother
+    # Default smoothing factor (default is 1%, but as our "confidence" increases we decrease the alpha)
```

Comment:
if I understand the code correctly, your plan is to eventually decrease it to 0.0001, so the comment is incorrect